.github/workflows/pylint.yml

```diff
@@ -0,0 +1,27 @@
+name: pylint
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  pylint_check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 2.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 2.7
```

Comment:
Is it possible to run the checks on both 2.6 and 2.7? Because RHEL 6 has py2.6.

Reply:
I'll check it out